ccmlib/scylla_node.py

```diff
@@ -1404,6 +1404,14 @@ def run_scylla_sstable(self, command, additional_args=None, keyspace=None, dataf
         """
         if additional_args is None:
             additional_args = []
+
+        additional_args += ['--scylla-yaml-file', str(Path(self.get_conf_dir()) / common.SCYLLA_CONF)]
+        if keyspace:
+            additional_args += ['--keyspace', keyspace]
+
+        if column_families and len(column_families) == 1:
+            additional_args += ['--table', column_families[0]]
+
```

Comment:
i'd suggest create yet another wrapper method around `run_scylla_sstable()` and pass `addition_args` to with the expected argument instead, like

```py
def dump_sstable_scylla_metadata(self, keyspace, column_family):
    # ...
    additional_args = ['--keyspace', keyspace,
                       '--table', column_family]
    scylla_metadata = self.run_scylla_sstable('dump-scylla-metadata',
                                               additional_args,
                                               keyspace=keyspace,
                                               column_families=[column_family])
```

Reply:
including the yaml file ?

Reply:
I've moved it to a new function, to minimize the changes to the other functions